plaso/parsers/sqlite_plugins/mac_notificationcenter.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+"""This file contains a parser for the Notification Center database on MacOS.
+
+Notification Center events on MacOS are stored in a
+SQLite database file named "db", path is usually something like
+/private/var/folders/<W><d>/../0/com.apple.notificationcenter/db2/
+
+At the moment it takes into consideration only the main table, 'record'.
+Documentation of the behavior of each table still work in progress,
+current tables and supposed related content is the following:
+    Record: contains Historical Records
+    Requests: contain pending requests
+    Delivered: delivered requests
+    Displayed: Displayed requests, by app_id
+    Snoozed: Snoozed by user requests
+"""
+
+from __future__ import unicode_literals
+
+from dfdatetime import cocoa_time as dfdatetime_cocoa_time
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.parsers import sqlite
+from plaso.parsers.sqlite_plugins import interface
+from biplist import *
```

Comment:
Don't do this - import biplist instead. See the style guide: https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports

Reply:
done